bundle/bundle.go

```diff
@@ -39,6 +39,12 @@ type Bundle struct {
 	// Exclusively use this field for filesystem operations.
 	BundleRoot vfs.Path
 
+	// SyncRoot is a virtual filesystem path to the root of the files that
+	// are synchronized to the workspace. It can be an ancestor to [BundleRoot],
+	// but not a descendant; that is, [SyncRoot] must contain [BundleRoot].
+	SyncRoot     vfs.Path
+	SyncRootPath string
```

Comment:
SyncRootPath is the remote path were files will be synced to, correct? The naming seems to be a bit confusing when I read the rest of the code

Reply:
I added a comment to clarify; this is the local path to the sync root (the same as `SyncRoot.Native()`).